components/slices/banner.vue

```diff
@@ -0,0 +1,231 @@
+<template>
+  <section :class="`${bannerWithBackgroundImageClass}`">
```

Comment:
Ne peut-on pas faire comme la `div` juste au-dessous ?
```suggestion
  <section :class="{ 'banner-with-background': hasBackgroundImage }">
```
Et du coup remplacer la propriété computed `bannerWithBackgroundImageClass` par `hasBackgroundImage`.